From 87502e5868a8dbe2fda9a8e66a37ed6996646ef8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 30 Aug 2014 22:33:40 -0400
Subject: [PATCH] units: make emergency.service conflict with rescue.service

They both use StandardInput=tty-force so they cannot be run
concurrently.

https://bugs.freedesktop.org/show_bug.cgi?id=82778
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=757072
---
 units/emergency.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/emergency.service.in b/units/emergency.service.in
index 91fc1bbf51..224a434f72 100644
--- a/units/emergency.service.in
+++ b/units/emergency.service.in
@@ -10,6 +10,7 @@ Description=Emergency Shell
 Documentation=man:sulogin(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
+Conflicts=rescue.service
 Before=shutdown.target
 
 [Service]
